src/patternfly/components/Expandable/expandable.scss

```diff
@@ -0,0 +1,84 @@
+@import "../../patternfly-imports";
+
+.pf-c-expandable {
+  // Expandable
+  --pf-c-expandable--Width: #{pf-size-prem(450px)};
```

Comment:
We can remove the width (so it's auto/100% by default) and let the parent element control the width.